### PR TITLE
added since and tail log queries

### PR DESCRIPTION
**Pull request by antonjlin**

### CODE REVIEW

**Comment by kuujo** [comment]
Needs to be rebased

**Comment by Andrea-Campanella** [comment]
@antonjlin this needs rebasing. Also travis did not trigger for some reason, if the rebase does not trigger it we can look into it.

**Comment by kuujo** [comment]
#431

File: test/cli/cli.go [hunk 1]
```diff
@@ -699,6 +708,12 @@ To output the logs from a test, get the test ID from the test run or from 'onit
 		},
 	}
 
+	cmd.Flags().Int64P("since", "", -1, "Only return logs newer than a relative " +
```
Comment (kuujo) [inline test/cli/cli.go]: Cobra supports a `Duration` type that may actually parse string input like `5s` or `10m` like the comment suggests? Currently this only takes a numeric input right?

Comment (antonjlin) [reply test/cli/cli.go]: Added in the latest commit.

File: test/cli/cli.go [hunk 2]
```diff
@@ -707,6 +722,26 @@ To output the logs from a test, get the test ID from the test run or from 'onit
...
+		options.SinceSeconds = &sinceSeconds
+	}
+	if err != nil {
+		exitError(err)
+	}
+	tail, err := cmd.Flags().GetInt64("tail")
+	if tail > 0 {
+		options.TailLines = &tail
+	}
+	fmt.Println(options)
```
Comment (kuujo) [inline test/cli/cli.go]: Need to remove the options before this is merged

**Pull request merged**